### PR TITLE
UI: fix activity card alignment with resource prov. card

### PR DESCRIPTION
**Pull request by alfonsomthd**
Screenshot from 2022-08-09 09-23-49

Signed-off-by: Alfonso Martínez <almartin@redhat.com>

### CODE REVIEW

**Comment by alfonsomthd** [comment]
/testci/prow/mcg-ms-console-e2e-aws

File: packages/components/dashboard/dashboard.scss [hunk 1]
```diff
@@ -18,6 +18,10 @@
   display: flex;
 }
 
+.pf-c-card {
+  height: 100%;
+}
+
```
Comment (SanjalKatiyar) [inline packages/components/dashboard/dashboard.scss:24]: which component has this class applied to it ?
are we overriding default class of any pf component ?

**Comment by alfonsomthd** [comment]
@SanjalKatiyar Right, I prefixed it with `.co-dashboard-body` so it only applies to the Overview cards.

File: packages/components/dashboard/dashboard.scss [hunk 2]
```diff
@@ -18,6 +18,10 @@
   display: flex;
 }
 
+.co-dashboard-body .pf-c-card {
```
Comment (SanjalKatiyar) [inline packages/components/dashboard/dashboard.scss:21]: I am not aware of this format (2 class names with a space in between), is it a form of descendant combinator ??

**Comment by SanjalKatiyar** [comment]
> @SanjalKatiyar Right, I prefixed it with `.co-dashboard-body` so it only applies to the Overview cards.

great.

**Pull request merged**